src/components/CMSGuidesNav.astro

```diff
@@ -0,0 +1,61 @@
+---
+import { getLanguageFromURL } from '../util';
+import CardsNav from './NavGrid/CardsNav.astro';
+
+interface Frontmatter {
+	title: string;
+	service: string;
+	stub: boolean;
+}
+
+const lang = getLanguageFromURL(Astro.url.pathname);
+
+const pages = await Astro.glob<Frontmatter>('../pages/en/guides/cms/*.md');
+
+/** Array of services we have good content for and want to show first in the list. */
+const showFirst = ['Storyblok', 'Contentful', 'ButterCMS'];
```

Comment:
We can update this array if we ever want to move specific pages up to the top of the nav list.

Reply:
Perfect!